Guided Investigation - WAF data.ipynb

```diff
@@ -0,0 +1,4156 @@
+{
```

Comment:
<p>If this is run in Azure Sentinel/Azure ML with msticpy - you can the workspace, tenantID from config.json or msticpyconfig.yaml</p><p>Look at some of the other notebooks - use WorkspaceConfig (which you import) and use ws_config.code_connection_st to pass to qry_prov.connect.</p><p> </p><p>Not a big deal but a bit neater and less UI.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>